### ISSUE TITLE
New IoT Edge solution fails with "ENOENT: no such file or directory"

### ISSUE THREAD

**Issue by rido-min** [microsoft/vscode-azure-iot-edge#583]
Version: 1.57.1 (user setup)
Commit: 507ce72a4466fbb27b715c3722558bb15afa9f48
Date: 2021-06-17T13:28:07.755Z (2 wks ago)
Electron: 12.0.7
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.19043

Steps to Reproduce:

1. Install this extension
2. F1-> New IoT Edge Solution

Expected. The solution is created
Observed: 
```
The specified iotedgehubdev version is: 0.14.8
Error: ENOENT: no such file or directory, lstat 'c:\Users\rmpab\.vscode\extensions\vsciot-vscode.azure-iot-edge-1.24.1\assets\solution\.gitignore'
```

**Comment by rido-min** [comment]
fixed by adding the missing `.gitignore` file

**Comment by vadim-kovalyov** [comment]
I have the same problem. Thanks @rido-min for suggesting the workaround. I believe it still needs to be fixed by the extension maintainers.

```
ENOENT: no such file or directory, lstat '/home/vakavali/.vscode/extensions/vsciot-vscode.azure-iot-edge-1.24.1/assets/solution/.gitignore'
```

**Comment by mhshami01** [comment]
Hello @rido-min, Thank you for bringing this to our attention. I have added an item to our backlog for tracking.

**Comment by RotateAt60MPH** [comment]
For the short term, you can create a temporary .gitignore file and put it in: .vscode\extensions\vsciot-vscode.azure-iot-edge-1.24.1\assets\solution\.gitignore'

Contents: 

```
CMakeCache.txt
CMakeFiles
CMakeScripts
Testing
Makefile
cmake_install.cmake
install_manifest.txt
compile_commands.json
CTestTestfile.cmake

```

**Comment by marianan** [comment]
The issue is resolved in 1.24.2 version of VSCode extension. @rido-min could you please try and confirm if you still see the original symptoms? Thanks

**Comment by mohseen18** [comment]
> fixed by adding the missing `.gitignore` file

Can you please let me know how did you do it?
